ext/standaloneusers/settings/standaloneusers.setting.php

```diff
@@ -16,13 +16,19 @@
   // Example value: 'TOTP', not 'Civi\Standalone\MFA\TOTP'
   'standalone_mfa_enabled' => [
     'name'        => 'standalone_mfa_enabled',
+    'serialize'   => CRM_Core_DAO::SERIALIZE_JSON,
     'group'       => 'standaloneusers',
     'type'        => 'String',
     'title'       => ts('Multi-Factor Authentication classes'),
-    'description' => ts('Comma separated list of classes of MFA that are required/accepted.'),
+    'description' => ts('Comma separated list of classes of MFA that are required/accepted. Leave blank to disable MFA. TOTP is Time-based One-Time Password which requires an authenticator app to provide a code.'),
```

Comment:
The message is a bit confusing in the UI settings page. Would suggest:

```suggestion
    'description' => ts('Choose which multi-factor options are required/accepted. Leave blank to disable MFA. TOTP is Time-based One-Time Password which requires an authenticator app to provide a code.'),
```